yarn.lock

```diff
@@ -2238,7 +2238,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@ibm-watson/discovery-react-components@^3.1.0, @ibm-watson/discovery-react-components@workspace:packages/discovery-react-components":
+"@ibm-watson/discovery-react-components@^3.1.1, @ibm-watson/discovery-react-components@workspace:packages/discovery-react-components":
```

Comment:
This version doesn't exist any more.

Reply:
I've tried to get the checks to pass without this, but I haven't been able to...

```
Post-resolution validation
  ➤ YN0000: │ @@ -2237,9 +2237,9 @@
  ➤ YN0000: │    checksum: a824a1ec31591231e4bad5787641f59e9633827d0a2eaae131a288d33c9ef0290bd16fda8da6f7c0fcb014147865d12118df10db57f27f41e20da92369fcb3f1
  ➤ YN0000: │    languageName: node
  ➤ YN0000: │    linkType: hard
  ➤ YN0000: │  
  ➤ YN0028: │ -"@ibm-watson/discovery-react-components@^3.1.0, @ibm-watson/discovery-react-components@workspace:packages/discovery-react-components":
  ➤ YN0028: │ +"@ibm-watson/discovery-react-components@^3.1.1, @ibm-watson/discovery-react-components@workspace:packages/discovery-react-components":
  ➤ YN0000: │    version: 0.0.0-use.local
  ➤ YN0000: │    resolution: "@ibm-watson/discovery-react-components@workspace:packages/discovery-react-components"
  ➤ YN0000: │    dependencies:
  ➤ YN0000: │      "@react-hook/size": ^2.1.2
  ➤ YN0000: │ @@ -11092,9 +11092,9 @@
  ➤ YN0000: │    version: 0.0.0-use.local
  ➤ YN0000: │    resolution: "discovery-search-app@workspace:examples/discovery-search-app"
  ➤ YN0000: │    dependencies:
  ➤ YN0000: │      "@carbon/icons": ^10.5.0
  ➤ YN0028: │ -    "@ibm-watson/discovery-react-components": ^3.1.0
  ➤ YN0028: │ +    "@ibm-watson/discovery-react-components": ^3.1.1
  ➤ YN0000: │      "@ibm-watson/discovery-styles": ^3.0.7
  ➤ YN0000: │      "@testing-library/cypress": ^7.0.7
  ➤ YN0000: │      "@types/proper-url-join": ^2
  ➤ YN0000: │      body-parser: ^1.19.0
  ➤ YN0000: │ 
  ➤ YN0028: │ The lockfile would have been modified by this install, which is explicitly forbidden.
➤ YN0000: └ Completed
➤ YN0000: Failed with errors in 0s 8[74](https://github.com/watson-developer-cloud/discovery-components/actions/runs/3187101553/jobs/5198343822#step:5:76)ms
```

https://github.com/watson-developer-cloud/discovery-components/actions/runs/3187101553/jobs/5198343822

Reply:
Any ideas? Maybe remove the `3.1.1` commit from `master`?